integ/src/main.rs

```diff
@@ -20,6 +20,11 @@ const DEFAULT_KUBECONFIG_FILE_NAME: &str = "kubeconfig.yaml";
 const DEFAULT_REGION: &str = "us-west-2";
 const CLUSTER_NAME: &str = "brupop-integration-test";
 
+//The default values for AMI ID
+const AMI_ARCH: &str = "x86_64";
+const BOTTLEROCKET_VERSION: &str = "1.6.1";
+const EKS_VERSION: &str = "1.21";
```

Comment:
Not sure if we should default those values here since we might need maintain them periodically.

Reply:
I agree that maybe the `BOTTLEROCKET_VERSION` shouldn't have a default. But we may want to ensure that the architecture and EKS versions match the cluster that can be generated?

Reply:
Yeah. I'll different strategies for those variables.
`BOTTLEROCKET_VERSION`: No default, completely provided by user.
`AMI_ARCH ` : Default to `x86_64`, and `arm64` specified by user.
`EKS_VERSION `:  Default to the version that matches the cluster (might be able to be generated), leave a option for user to specify if they want to test with other EKS version.